wagtail/core/models.py

```diff
@@ -192,22 +192,22 @@ def get_queryset(self):
 
 class PageBase(models.base.ModelBase):
     """Metaclass for Page"""
-    def __init__(cls, name, bases, dct):
-        super(PageBase, cls).__init__(name, bases, dct)
+    def __init__(cls, name, bases, dct, **kwargs):
+        super(PageBase, cls).__init__(name, bases, dct, **kwargs)
```

Comment:
While you're at it, you might as well remove `PageBase, cls` in `super` since we do not support Python 2.x anymore.